airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -113,6 +113,16 @@ public ConnectionManagerWorkflowImpl() {}
   public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws RetryableException {
     try {
       try {
+        connectionId = connectionUpdaterInput.getConnectionId();
```

Comment:
Some of the logic in `() -> skipScheduling()` depends on the values in `workflowState` being set correctly, so moving that here before `workflowState` is properly set up will likely lead to issues. For example, `skipScheduling()` looks at `workflowState.isResetConnection()` as part of its logic, so this needs to be set from the inputs beforehand. And `skipScheduling()` needs to depend on `workflowState` rather than `connectionUpdaterInput`, because `workflowState` is what gets modified in signal methods like starting a manual sync, and the `skipScheduling()` logic needs to react to those signals.